microsoft-edge/webview2/releasenotes.md

```diff
@@ -12,29 +12,66 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
 
 # Release notes for WebView2 SDK  
 
-The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a 6-week cadence.  Review the following content for up to date information about product announcements, additions and modifications to the API surface, and breaking changes.  
+The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a six week cadence.  Review the following content for up-to-date information about product announcements, additions and modifications to the API surface, and breaking changes.  
 
 > [!NOTE]
 > Re-compile your app after updating the NuGet package.  
 
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `pre-release package`.  
 
+
+## 0.9.628-prerelease  
+Release Date: Sept 09, 2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+   > [!IMPORTANT]
+    > **Announcement**: This prerelease will continue to release out of Edge 87 Branch. In the future, pre-release SDK releases will match the Edge Canary builds, whereas our normal releases will be in sync with Edge Stable.
+#### General
+* Added the CompositionController as an experimental interface. This lets your app connect WebView2 to Dcomp visual or WinComp ContainerVisual and provides API that allows the app to forward mouse and touch inputs to the WebView.
+
+#### .NET  
+
+*   .NET Binaries are now [Strongly Named]().  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).
+*   Updated NuGet Target to include WebViewLoader.dll.  \([\#228][GithubMicrosoftedgeWebviewfeedbackIssue228]\) & \([\#183][GithubMicrosoftedgeWebviewfeedbackIssue183]\).
+*   Updated WebResourceRequested to expose HttpRequestMessage and HttpResponseMessage APIs in .NET.  \([\#131][GithubMicrosoftedgeWebviewfeedbackIssue131]\).
+
+
+## 0.9.622  
+Release Date: Sept 09, 2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**: This SDK release is a candidate for WebView2 Win32 C++ GA. 
```

Comment:
I would say "SDK 0.9.622 is the release candidate for WebView2 Win32 C/C++ GA. The GA version will likely have the same API interface and functionality."